spec/use_cases/schema2.0.0/BRICR_SEED.sch

```diff
@@ -9,13 +9,13 @@
       <sch:assert test="count(./auc:Buildings/auc:Building/auc:Address/auc:PostalCode) = 1">[ERROR] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:Address/auc:PostalCode" is REQUIRED</sch:assert>
-      <sch:assert test="count(./auc:Buildings/auc:Building/auc:Longitude) = 1">[ERROR] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:Longitude" is REQUIRED</sch:assert>
-      <sch:assert test="count(./auc:Buildings/auc:Building/auc:Latitude) = 1">[ERROR] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:Latitude" is REQUIRED</sch:assert>
+      <sch:assert test="count(./auc:Buildings/auc:Building/auc:Longitude) = 1">[WARNING] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:Longitude" is REQUIRED</sch:assert>
+      <sch:assert test="count(./auc:Buildings/auc:Building/auc:Latitude) = 1">[WARNING] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:Latitude" is REQUIRED</sch:assert>
       <sch:assert test="count(./auc:Buildings/auc:Building) = 1">[ERROR] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building" is REQUIRED</sch:assert>
       <sch:assert test="count(./auc:Buildings/auc:Building/auc:Sections/auc:Section/auc:OccupancyClassification) = 1">[ERROR] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:Sections/auc:Section/auc:OccupancyClassification" is REQUIRED</sch:assert>
       <sch:assert test="count(./auc:Buildings/auc:Building/auc:YearOfConstruction) = 1">[ERROR] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:YearOfConstruction" is REQUIRED</sch:assert>
-      <sch:assert test="count(./auc:Buildings/auc:Building/auc:ConditionedFloorsAboveGrade) = 1">[ERROR] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:ConditionedFloorsAboveGrade" is REQUIRED</sch:assert>
-      <sch:assert test="count(./auc:Buildings/auc:Building/auc:ConditionedFloorsBelowGrade) = 1">[ERROR] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:ConditionedFloorsBelowGrade" is REQUIRED</sch:assert>
+      <sch:assert test="count(./auc:Buildings/auc:Building/auc:ConditionedFloorsAboveGrade) = 1">[WARNING] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:ConditionedFloorsAboveGrade" is REQUIRED</sch:assert>
+      <sch:assert test="count(./auc:Buildings/auc:Building/auc:ConditionedFloorsBelowGrade) = 1">[WARNING] element "/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building/auc:ConditionedFloorsBelowGrade" is REQUIRED</sch:assert>
```

Comment:
For all `WARNING` levels, we always use `RECOMMENDED` at the end instead of `REQUIRED`